src/App.css

```diff
@@ -1,28 +1,179 @@
-.app {
-  height: 100vh;
-  width: 100vw;
-  transition: background-color 1s ease-in;
+.app__header {
+  background-color: dodgerblue;
+  color: white;
+  display: flex;
+  height: 80px;
 }
-
-.app__header, 
-.app__main {
+.currentWeather,
+.Forcast {
+  height: 100%;
   width: 100%;
+  transition: background-color 1s ease-in;
+  background-color: lightskyblue;
+  font-family: "Raleway";
   display: flex;
   flex-direction: column;
   align-items: center;
   padding: 20px;
-  font-family: 'Raleway', sans-serif;
+  font-family: "Raleway", sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
 
-
 /*  Search bar */
 
-
+.btn {
+  color: darkblue;
+  border: 1px solid rgb(0, 102, 255);
+  background-color: rgb(0, 102, 255);
+  padding: 10px 30px 10px 30px;
+  margin: 15px;
+  position: relative;
+  left: 200px;
+  font-size: 15px;
+  display: inline;
+}
+.text {
+  background: transparent;
+  border: none;
+  color: white;
+  font-size: 20px;
+  outline: none;
+  position: relative;
+  left: 15px;
+}
+.text:focus {
+  border-bottom: 1px solid lightskyblue;
+}
+::placeholder {
+  color: white;
+}
 
 /* Current weather */
-
-
-
-
+.main-img {
+  width: 25%;
+}
+.flex-container1 {
+  display: flex;
+  flex-direction: column;
+}
+.title {
+  text-align: center;
+  color: white;
+}
+.temp {
+  color: darkblue;
+  font-size: 20px;
+}
+.tempInfo {
+  color: darkblue;
+  margin: 5px;
+  font-size: 13px;
+}
+.flex-container {
+  display: flex;
+  color: darkblue;
+  flex-direction: row;
+  width: 75%;
+  text-align: center;
+}
+.flex-box1 {
+  justify-content: space-around;
+}
+.flex-box2 {
+  justify-content: space-around;
+}
+.flex-box3 {
+  justify-content: space-around;
+}
+.flex-box5 {
+  justify-content: space-around;
+}
+.flex-box6 {
+  justify-content: space-around;
+}
+.flex-box7 {
+  justify-content: space-around;
+}
+.img-size {
+  width: 40%;
+}
+.Forcast {
+  display: inline-flexbox;
+}
+@media (max-width: 576px) {
+  .flex-container1 {
+    display: flex;
+    flex-direction: column;
+  }
+  .flex-container {
+    display: flex;
+    flex-direction: column;
+  }
+  .tempInfo {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+  }
+  .text {
+    position: relative;
+    left: 50px;
+  }
+  .btn {
+    position: relative;
+    left: 50px;
+  }
+  @media (max-width: 768px) {
+    .flex-container1 {
+      display: flex;
+      flex-direction: column;
+    }
+    .tempInfo {
+      display: flex;
+      flex-direction: column;
+      text-align: center;
+    }
+    .flex-container {
+      display: flex;
+      flex-direction: column;
+    }
+    .tempInfo {
+      display: flex;
+      flex-direction: column;
+      text-align: center;
+    }
+    .text {
+      position: relative;
+      left: 50px;
+    }
+    .btn {
+      position: relative;
```

Comment:
You don't need to repeat rules in media queries that you have already declared outside of media queries. For example:
this `.btn` class is already set to `position: relative` on line 31, so you don't need to repeat that. Only include in media queries what you want to change at a different screen size.